modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1855,7 +1872,16 @@ private void positionCell(T cell, double position) {
         }
     }
 
-    private void resizeCellSize(T cell) {
+    /**
+     * Resizes the given cell. If {@link #isVertical()} is set to {@code true}, the cell width will be the maximum
+     * between the viewport width and the sum of all the cells' {@code prefWidth}. The cell height will be computed by
+     * the cell itself unless {@code fixedCellSizeEnabled} is set to {@code true}, then {@link #getFixedCellSize()} is
+     * used. If {@link #isVertical()} is set to {@code false}, the width and height calculations are reversed.
+     *
+     * @param cell the cell to resize
+     * @since 12
+     */
+    protected void resizeCellSize(T cell) {
```

Comment:
I think `resizeCell` would be a better name for this method. The `Size` suffix seems redundant.